src/spetlr/delta/delta_handle.py

```diff
@@ -25,22 +28,60 @@ class DeltaHandleInvalidFormat(DeltaHandleException):
 
 
 class DeltaHandle(TableHandle):
-    def __init__(self, name: str, location: str = None, data_format: str = "delta"):
+    def __init__(
+        self,
+        name: str,
+        location: str = None,
+        data_format: str = "delta",
+        options_dict: Dict[str, str] = None,
+        ignore_changes: bool = True,
+        stream_start: Union[datetime, str] = None,
+        max_bytes_per_trigger: int = None,
+    ):
+        """
+        name: The name of the Delta table.
+        location (optional): The file-system path to the Delta table files.
+        data_format (optional): Always delta-format. Todo: Remove in future PR.
+        options_dict (optional): All other string options for pyspark.
+        ignore_changes (optional): ignore transactions that delete data
+                                    at partition boundaries.
+        stream_start (optional):  If string format, it accepts anything that
+                                  the `dateparser` library can parse.
+        max_bytes_per_trigger (optional): How much data gets
+                                processed in each micro-batch.
+        """
         self._name = name
         self._location = location
         self._data_format = data_format
 
         self._partitioning: Optional[List[str]] = None
-
         self._validate()
 
+        if options_dict is None or options_dict == "":
+            self.set_options_dict({})
+        else:
+            self.set_options_dict(options_dict)
+
+        self._options_dict.update({"ignoreChanges": str(ignore_changes)})
+
+        if stream_start and stream_start != "":
+            self._options_dict["startingTimestamp"] = stream_start.strftime(
+                "%Y-%m-%dT%H:%M:%S.%fZ"
```

Comment:
The startingTimestamp need a specific format: %Y-%m-%dT%H:%M:%S.%fZ
How is this handled when read from table config.
Should we validate the format?